package.json

```diff
@@ -6,7 +6,8 @@
   "repository": "mozilla/normandy-actions",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "./node_modules/.bin/webpack --progress",
+    "build": "./node_modules/.bin/webpack --progress --colors",
+    "watch": "./node_modules/.bin/babel-node ./bin/watch.js",
```

Comment:
Some of the task cluster people yesterday were ranting about how surprisingly bad babel-node was. Maybe we should use something else eventually.